packages/core/src/render3/instructions.ts

```diff
@@ -2214,12 +2214,13 @@ export function createLContainer(
   ngDevMode && assertDomNode(native);
   ngDevMode && assertLView(currentView);
   return [
+    hostNative,  // host native
+    true,
```

Comment:
Nit: add comment for what `true` signifies?

```suggestion
    true,     // type
```

Reply:
added comment.